pkg/reconciler/knservice/resources/knative_service.go

```diff
@@ -0,0 +1,54 @@
+package resources
+
+import (
+	knservingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha1"
+	"github.com/kubeflow/kfserving/pkg/containers/tensorflow"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func getModelServingContainer(modelSpec *v1alpha1.ModelSpec, modelName string) *v1.Container {
```

Comment:
nit: spec, name -> name, spec. This is more familiar to most libraries I'm familiar with that do key, value. Perhaps this is a go convention?

Also, consider CreateContainerSpec for parity with CreateTensorflowContainer